aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -908,7 +916,11 @@ _scaled_mm_out_cuda(const Tensor& mat1, const Tensor& mat2,
       bias ? bias->data_ptr(): nullptr,
       bias ? bias->scalar_type() : isFloat8Type(out_dtype_) ? at::ScalarType::Half : out_dtype_,
       args.result->data_ptr(),
+#if !defined(USE_ROCM)
       scale_result ? scale_result->data_ptr() : nullptr,
+#elif defined(USE_ROCM) && ROCM_VERSION >= 60200
+      scale_result ? scale_result->data_ptr() : dummy_scale.data_ptr(),
```

Comment:
As I understand scaleD(scale_result) cant be NULL pointer for hipBlasLT, but ok for cuBlasLT. Should we raise an issue to hipBlasLT to check on this difference in behavior?